Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2227,17 +2227,24 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         return topicGraph.reverseEdges[reference] ?? []
     }
     
-    /// Attempt to locate the file for a given `reference`.
+    /// Returns the file URL for the given article reference.
```

Comment:
I'd think that this implementation would work for tutorial references as well. Is this API only meant to work for articles?

Reply:
Good point. I added a check to ensure the reference is to a symbol.